### ISSUE TITLE
do not hard code the kibella web installation dir

### ISSUE THREAD

**Issue by Kranf99** [Kranf99/kibella#59]
inside the file "interface/kibana.js", we can read:

```
  var secondSlash = window.location.pathname.indexOf('/', 1);
  var kbnPath = window.location.pathname.substr(0, secondSlash) || '/kibella';
```
This means that the value of kbnPath is "hard coded" to a directory named "kibella".
This is not good at all!

**Comment by Kranf99** [comment]
I just made a small commit today about that: see here:
https://github.com/Kranf99/kibella/commit/cbc8cfddab3be71d023cac129188ee900557a8a6

Basically, this commit adds some lines of code inside the file "interface\node_modules\elasticsearch-browser": These are the added lines:
```
	  if ((typeof config === 'string')&&(config.indexOf("..")===0)) {
		  this.protocol = '';
		  this.host = '';
		  var s=window.location.pathname;
		  s=s.substring(0,s.lastIndexOf("/"));
		  s=s.substring(0,s.lastIndexOf("/"));
		  if (s.charAt(s.length-1)=='/') s=s.substring(0,s.length-1);
		  this.path = s+config.substring(2);
		  this.port = 0;
		  this.query = null;
		  this.headers = null;
		  this.suggestCompression = !!globalConfig.suggestCompression;
		  this.ssl=null;
	  	  return;
	  }
```

The above code is inserted just after:
```
 	function Host(config, globalConfig) {
 	  config = _.clone(config || {});
 	  globalConfig = globalConfig || {};
```

This is not enough to allow to install Kibella in any directory, but it's a start

**Comment by Kranf99** [comment]
When this issue is solved, it should also (normally) solve issue #66.

**Comment by Kranf99** [comment]
It was actually already working before.
No need to worry! ;-)
I reverted the "fix" from Leo since it's useless.
